ingest-app/docs/api.md

```diff
@@ -741,7 +736,7 @@ Tool concept can be created or updated by sending an HTTP PUT with the metadata
 ```
 curl -i -XPUT \
   -H "Content-type: application/vnd.nasa.cmr.umm+json" \
-  -H "Echo-Token: XXXX" \
+  -H "Authorization: Bearer XXXX" \
   %CMR-ENDPOINT%/providers/PROV1/tools/tool123 \
   -d \
 "{\"Name\": \"USGS_TOOLS_LATLONG\", \"LongName\": \"WRS-2 Path/Row to Latitude/Longitude Converter\", \"Type\": \"Downloadable Tool\", \"Version\": \"1.0\", \"Description\": \"The USGS WRS-2 Path/Row to Latitude/Longitude Converter allows users to enter any Landsat path and row to get the nearest scene center latitude and longitude coordinates.\", \"URL\": { \"URLContentType\": \"DistributionURL\", \"Type\": \"DOWNLOAD SOFTWARE\", \"Description\": \"Access the WRS-2 Path/Row to Latitude/Longitude Converter.\", \"URLValue\": \"http://www.scp.byu.edu/software/slice_response/Xshape_temp.html\" }, \"ToolKeywords\" : [{ \"ToolCategory\": \"EARTH SCIENCE SERVICES\", \"ToolTopic\": \"DATA MANAGEMENT/DATA HANDLING\", \"ToolTerm\": \"DATA INTEROPERABILITY\", \"ToolSpecificTerm\": \"DATA REFORMATTING\" }], \"Organizations\" : [ { \"Roles\": [\"SERVICE PROVIDER\"], \"ShortName\": \"USGS/EROS\",    \"LongName\": \"US GEOLOGICAL SURVEY EARTH RESOURCE OBSERVATION AND SCIENCE (EROS) LANDSAT CUSTOMER SERVICES\", \"URLValue\": \"http://www.usgs.gov\" } ], \"MetadataSpecification\": { \"URL\": \"https://cdn.earthdata.nasa.gov/umm/tool/v1.0\", \"Name\": \"UMM-T\", \"Version\": \"1.0\" }"
```

Comment:
since your in the code, I think you should update the example json from 1.0 tool to 1.2. Continuing to give examples of outdated metadata may result in the generation of support emails. https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/tool/v1.2.0